README.md

```diff
@@ -0,0 +1,11 @@
+### Earthquake 
+
+
+
+![screenshot when it works] (![1hour.png](..%2F..%2FDesktop%2F1hour.png))
```

Comment:
The location of the image doesn't work

Reply:
It's still not working after clearing the caches so I put nothing there till I can get a screenshot

Reply:
Let's meet a few minutes before class.